### ISSUE TITLE
On HOLD; Material UI: Change JDI prefix to J in custom annotations

### ISSUE THREAD

**Issue by lyubavinaju** [jdi-testing/jdi-light#3961]
Change JDI prefix to J in custom annotations.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/45433937/160753012-0531abd2-3efc-46f2-9783-9068a8f7b452.png)

**Comment by Kratlis** [comment]
This issue was approved. The annotations have to have the prefix 'J'.

**Comment by tsvetaevAlex** [comment]
Task On HOLD due to:waiting for acknowledgement for acceptance team

**Comment by tsvetaevAlex** [comment]
Task and PR unblocked due to prefix J approved by Roman
